src/Features/Core/Portable/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer.cs

```diff
@@ -84,7 +84,7 @@ public bool ContainsDiagnostics(Workspace workspace, ProjectId projectId)
         public bool NeedsReanalysisOnOptionChanged(object sender, OptionChangedEventArgs e)
         {
             return e.Option.Feature == nameof(SimplificationOptions) ||
-                   e.Option.Feature == nameof(CodeStyleOptions) ||
+                   e.Option.Feature.EndsWith(nameof(CodeStyleOptions), StringComparison.OrdinalIgnoreCase) ||
```

Comment:
I guess we don't have any better way?

Reply:
There exists both a `CSharpCodeStyleOptions` and a `VisualBasicCodeStyleOptions` that we would want to include. The `CSharpCodeStyleOptions` can't be referenced with a `nameof()`. This would include all three CodeStyleOptions classes and I think have little chance of colliding unintentionally.